script/passenger-recycler.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+#
+# Trivial Passenger memory monitor installed via /etc/cron.hourly/. To modify
+# configuration create /etc/foreman/passenger-recycler.rb.conf file.
+#
+
+CONFIG = '/etc/foreman/passenger-recycler.rb.conf'
```

Comment:
That is indeed possible, do you insist on this? I find config file more clear, you can provide comments with explanations and in this case you would need to create wrapper or system wide variables.